srcpkgs/neard/files/neard/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+sv check dbus >/dev/null || exit 1
+[ -f conf ] && . ./conf
+exec /usr/libexec/nfc/neard ${ARGS:--n}
```

Comment:
If -n means don't fork to background, we should set it all the time even if ARGS is set and we usually call this variable OPTS.